healthtools/settings.py

```diff
@@ -44,6 +44,10 @@
 SMS_MTECH_PASS = os.getenv('HTOOLS_SMS_MTECH_PASS')
 SMS_MTECH_SHORTCODE = os.getenv('HTOOLS_SMS_MTECH_SHORTCODE', '22495')
 
+AFRICASTALKING = {
+    "SMS_AFRICASTALKING_USER": os.getenv("SMS_AFRICASTALKING_USER"),
+    "SMS_AFRICASTALKING_KEY": os.getenv("SMS_AFRICASTALKING_KEY"),
+}
```

Comment:
Similar to MTECH, some 3 things here:

1. Let's have these as their own variables
2. env variables must start with `HTOOLS_`
3. Use single quotes